extensions/wikia/AchievementsII/AchConfig.class.php

```diff
@@ -2,10 +2,10 @@
 
 class AchConfig {
 	private static $mLevels = array(
-		BADGE_LEVEL_BRONZE => array('name' => 'bronze', 'score' => 10),
-		BADGE_LEVEL_SILVER => array('name' => 'silver', 'score' => 50),
-		BADGE_LEVEL_GOLD => array('name' => 'gold', 'score' => 100),
-		BADGE_LEVEL_PLATINUM => array('name' => 'platinum', 'score' => 250)
+		BADGE_LEVEL_BRONZE => array( 'name' => 'bronze', 'score' => 10 ),
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "score" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)